README.md

```diff
@@ -121,7 +121,11 @@ We are _very_ happy that oCIS does not require a Contributor License Agreement (
 Infinite Scale is carefully internationalized so that everyone, no matter what language they speak, has a great experience. To achieve this we rely on the help of volunteer translators. If you want to help, you can find the project behind this link:
  [Transifex for ownCloud web](https://explore.transifex.com/owncloud-org/owncloud-web/).
```

Comment:
```suggestion
Infinite Scale is carefully internationalized so that everyone, no matter what language they speak, has a great experience. To achieve this we rely on the help of volunteer translators. If you want to help, you can find the projects behind the following links:
 [Transifex for ownCloud web](https://app.transifex.com/owncloud-org/owncloud-web/translate/) and [Transifex for ownCloud](https://app.transifex.com/owncloud-org/owncloud/translate/) (Select the resource by filtering for `ocis-`).
```
The Transifex URL's have changed and we need to add `ocis-userlog` and `ocis-notifications`.